pandasdmx/remote.py

```diff
@@ -87,7 +92,7 @@ def read(self, size=None):
 
         If the argument is omitted, :py:obj:`None`, or negative, reads and
         returns all data until EOF. If *tee* was provided to the constructor,
-        data is echoed to file.
+        data is echoed to file. file is not closed.

```

Comment:
Please use complete sentences in docstrings with proper capitalization: “The file is not closed.”